src/script/components/calling/fullscreenVideoCall.test.ts

```diff
@@ -104,14 +104,14 @@ describe('fullscreenVideoCall', () => {
     expect(fullscreenVideoCall.getVideoTimer().text()).toEqual('00:01');
   });
 
-  it.skip('has no active speaker toggle for calls with more less than 3 participants', () => {
+  it('has no active speaker toggle for calls with more less than 3 participants', () => {
```

Comment:
More less?

Reply:
this was skipped in https://github.com/wireapp/wire-webapp/pull/10617/files#diff-20a7dc4d3e426edfce2c483b3d20e1f4d094582a0d9fbfaf3eb87c474112777a
now with bringing back the active speakers button it should be back